test_programs/execution_failure/bigint_from_too_many_le_bytes/src/main.nr

```diff
@@ -0,0 +1,22 @@
+use std::bigint::{bn254_fq, BigInt};
+
+// TODO(https://github.com/noir-lang/noir/issues/5580): decide whether this is desired behavior
```

Comment:
This seems different than the solver tests? Where does this fail in the example? I see it says line 7 but it looks like the comment has fudged where that is.

Reply:
It's failing on line 7 of `std/cmp.nr`. 

Here's a fresh error message:
```bash
     Running `/Users/michaelklein/Coding/rust/noir/target/debug/nargo execute`
error: Assertion failed: 'Index out of bounds'
   ┌─ std/cmp.nr:45:34
   │
45 │             result &= self[i].eq(other[i]);
   │                                  --------
   │
   = Call stack:
     1. /Users/michaelklein/Coding/rust/noir/test_programs/execution_failure/bigint_from_too_many_le_bytes/src/main.nr:21:24
     2. std/cmp.nr:45:34

Failed assertion
```

Reply:
I was looking at the line in the original Noir test `bigint_from_too_many_le_bytes/src/main.nr:21:24`. I guess they are related due to having a big int with too many bytes.

Reply:
Ah, yeah